src/pages/profilePage.jsx

```diff
@@ -1,145 +1,180 @@
-import React, {useState} from 'react';
-import '../css/profilePage.css';
-import QCards from '../components/questionCards';
-import {storage, updateDisplayName} from '../hooks/databaseHooks';
+import React, { useState } from "react";
+import "../css/profilePage.css";
+import QCards from "../components/questionCards";
+import {
+  storage,
+  updateDisplayName,
+  updatePhotoUrl
+} from "../hooks/databaseHooks";
 
 function Profile(props) {
-
-    const allInputs = {
-        imgUrl: ''
-    };
-    const [imageAsFile, setImageAsFile] = useState('');
-    const [tempName, setTempName] = useState(props.displayName);
-    const [imageAsURL, setImageAsUrl] = useState(allInputs);
-    const [profileImageURL, setProfileImageURL] = useState('');
-
-
-    /*Code adapted from Tallan Groberg
+  const allInputs = {
```

Comment:
Is there a reason we have this as a dict instead of just a single value? I don't really see what the use of this variable is.

Reply:
Made these changes